test/context.js

```diff
@@ -116,17 +116,17 @@ describe("Contexts", function () {
     let affectedInstances = session.view(sessionSelector.info.affectedInstances);
     debug("affectedInstances: %o", affectedInstances);
 
-    let affectedAddresses = Object.keys(affectedInstances);
+    let affectedAddresses = Object.keys(affectedInstances).map(address => address.toLowerCase());
 
     assert.equal(2, affectedAddresses.length);
 
     assert.include(
-      affectedAddresses, outer.address,
+      affectedAddresses, outer.address.toLowerCase(),
       "OuterContract should be an affected address"
     );
 
     assert.include(
-      affectedAddresses, inner.address,
+      affectedAddresses, inner.address.toLowerCase(),
       "InnerContract should be an affected address"
```

Comment:
Expected to be able to remove all the address normalization but looks like `affectedAddresses` are still lower-case coming out. Not sure if that's a problem or not. CI failure for post web3 update: c58c93d1726f6efae46db6485c25548499e19cc7
```shell
AssertionError: InnerContract should be an affected address: expected [ Array(2) ] to include '0xF248a7B090E45FBAc267719a09aEeFeDB4b2cd40'
```